src/overlays/actors/ovl_En_Ma2/z_en_ma2.h

```diff
@@ -8,6 +8,29 @@ struct EnMa2;
 
 typedef void (*EnMa2ActionFunc)(struct EnMa2*, GlobalContext*);
 
+typedef enum {
+    /* 0x00 */ MALON_ADULT_LIMB_NONE,
+    /* 0x01 */ MALON_ADULT_ROOT_LIMB,
+    /* 0x02 */ MALON_ADULT_LOWER_CONTROL_LIMB,
+    /* 0x03 */ MALON_ADULT_LEFT_THIGH_LIMB,
+    /* 0x04 */ MALON_ADULT_LEFT_LEG_LIMB,
```

Comment:
same as en_ma3
I guess eventually we will have these in object headers commited to the repo to avoid duplicating these enums